qa/integration/specs/dlq_spec.rb

```diff
@@ -43,14 +43,20 @@
       es_service = @fixture.get_service("elasticsearch")
       es_client = es_service.get_client
       # Wait for es client to come up
-      sleep(15)
+      sleep(20)
```

Comment:
@robbavey I wonder if we could just busy wait here instead? We should be able to wrap `es_client.indices.refresh` in a `try(20) { es_client.indices.refresh }` block shouldn't we?
I'm mainly asking because I've also seen the block below that you set from `75` to `60` time out on Travis, so I'm a little hesitant on just making that shorter.